openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml

```diff
@@ -158,317 +275,1305 @@
+  narrative:
+    - key: a
+      text: |
+        To assist with this organizational control, documentation is being
+        planned. Progress can be tracked via:
+
+        https://issues.redhat.com/browse/CMP-418
+    - key: b
+      text: |
+        To assist with this organizational control, documentation is being
+        planned. Progress can be tracked via:
+
+        https://issues.redhat.com/browse/CMP-418
+    - key: c
+      text: |
+        The customer will be responsible for authorizing, monitoring, and
+        controlling the use of mobile code within the system in accordance
+        with organizational policies defined in SC-18(a) and SC-18(b).
+
+- control_key: SC-18 (1)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        This control reflects organizational procedures/policies, and is not
+        applicable to the configuration of OpenShift.
+
+- control_key: SC-18 (2)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        This control reflects organizational procedures/policies, and is not
+        applicable to the configuration of OpenShift.
+
+- control_key: SC-18 (3)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        This control reflects organizational procedures/policies, and is not
+        applicable to the configuration of OpenShift.
+
+- control_key: SC-18 (4)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        This control reflects organizational procedures/policies, and is not
+        applicable to the configuration of OpenShift.
+
+- control_key: SC-18 (5)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        This control reflects organizational procedures/policies, and is not
+        applicable to the configuration of OpenShift.
+
+- control_key: SC-19
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - key: a
+      text: |
+        This control reflects organizational procedures/policies, and is not
+        applicable to the configuration of OpenShift.
+    - key: b
+      text: |
+        The customer will be responsible for authorizing, monitoring, and
+        controlling the use of VoIP within the system in accordance with
+        organizational policies defined in SC-19(a).
+
+- control_key: SC-20
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - key: a
+      text: |
+        OpenShift has a built-in DNS so that services can be reached by
+        the service DNS as well as the service IP/port. OpenShift supports
+        split DNS by running SkyDNS on the master that answers DNS queries
+        for services.
```

Comment:
The SkyDNS references are from OCP3. We shouldn't carry them forward.

Reply:
Well, the only references to SkyDNS appear to be OCP 3. For example:

Reply:
Done